docs/current/sdk/elixir/756758-get-started.md

```diff
@@ -0,0 +1,87 @@
+---
+slug: /sdk/elixir/756758/get-started
+---
+
+# Get Started with the Dagger Elixir SDK
+
+## Introduction
+
+This tutorial teaches you the basic of using Dagger in Elixir. You will learn how to:
+
+- Install Elixir SDK
+- Create an Elixir CI task to test an application
+- Improve the Elixir CI tool to test the application against multiple Elixir and OTP versions
+
+## Requirements
+
+This tutorial assumes that:
+
+- You have a basic understanding of the Elixir programming language. If not, [read the Elixir learning](https://elixir-lang.org/learning.html).
+- You have an Elixir development environment with Elixir 1.14 or later and Erlang/OTP 25 or later. If not, install [Elixir](https://elixir-lang.org/install.html) and [Erlang/OTP](https://www.erlang.org/downloads).
+- You have Docker installed and running on the host system. If not, [install Docker](https://docs.docker.com/engine/install/).
+
+## Step 1: Create an Elixir project
+
+Create an Elixir project with `mix`:
+
+```shell
+mix new a_project
```

Comment:
I would call this project something more descriptive than `a_project`.

What are your thoughts @wingyplus @vikram-dagger on `elixir-with-dagger` or `elixir_with_dagger` instead?

Reply:
Either of those is good with me

Reply:
I will be going into the `elixir_with_dagger` way. :)